mkosi-initrd/mkosi.finalize

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import os
+import pathlib
+import subprocess
+
+def buildroot():
+    return pathlib.Path(os.getenv('BUILDROOT'))
+
+def mask_units(root):
+    units = [
+        # Dracut installs a rule which sets OPTIONS+="db_persist" on all dm devices [1]
+        # (According to codesearch.debian.net, it is the only user of db_persist.)
+        # Without this, all dm units end up with SYSTEMD_READY=0 and systemd thinks the
+        # device units are missing. The system boots fine, but an attempt to call
+        # daemon-reexec or daemon-reload ends with anything that can be stopped or unmounted
+        # being purged.
```

Comment:
This is confusing to me. Are you saying that if the `db_persist` option is set, the system boots fine but daemon-reexec or daemon-reload end up purging all the lvm stuff?